packages/@okta/vuepress-theme-prose/index.js

```diff
@@ -14,9 +14,10 @@ module.exports = (options, ctx) => {
       $page.fullHeaders = resolveHeaders($page);
 
       // Set to true to enable the redesign in development
-      const redesign = process.env.REDESIGN || false;
+      // const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      // $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true;
```

Comment:
🚩 Just flagging with a comment to help remind us to revert this before merging

Reply:
Removed